lua/lspconfig/volar.lua

```diff
@@ -0,0 +1,103 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local function get_typescript_server_path(root_dir)
+  local project_root = util.find_node_modules_ancestor(root_dir);
+  return project_root and (project_root .. '/node_modules/typescript/lib/tsserverlibrary.js') or '';
+end
+
+local server_name = 'volar'
+local bin_name = 'volar-server'
+
+-- https://github.com/johnsoncodehk/volar/blob/master/packages/shared/src/types.ts
+local volar_init_options = {
```

Comment:
Do all of these options need to be explicitly set, or are these overriding defaults?

Reply:
Yes, need explicitly set to enable these features. Otherwise features was disabled.

Reply:
Thanks! If you could, since you are the expert, please let me know if anything needs to change in this PR regarding settings or init_options.

Reply:
The options look good to me! And no need to specifically set false for unsupported features, it will just ignore if unsupported.

Reply:
Just wanted to be explicit